Source/Documentation/Manual/news.rst

```diff
@@ -15,6 +15,7 @@ What's been added
 -----------------
```

Comment:
Didn't realize I never added a line here with the lighting updates. The new lighting features have proven to be popular with many creators eager to implement them, but I've seen people not realize the features are there until randomly looking through the manual-not ideal! Would help if there was a news entry for awareness there.

Maybe just...
- new rolling stock lighting controls, including horn-activated flashing lights

Reply:
Thanks, Phillip.

I've added that in now.

Best wishes,

Chris